ferdi ardiyansyah_211011401217/tugas_2.txt

```diff
@@ -0,0 +1,17 @@
+Ferdi Ardiyansyah_211011401217
+Functional
+Meja:
+Pelanggan bisa memilih menu
+Menentukan jumlah pesanan
+Memilih jenis pembayaran
+Memilih dine in atau take home
+Koki:
+Memberikan informasi tersedianya menu
+Memberi tahu stok makanan 
+
```

Comment:
Good job, but you can do more than this. Masih banyak yang bisa digali dan dideskripsikan terkait dengan fungsionalitas dan non-fungsionalitas dari yang diinginkan oleh user.